### PR TITLE
Adds deploy button

### PR DESCRIPTION
**Pull request by jonnilundy**
This project is amazing!! I hope this makes it even easier for others to use it. :)

### CODE REVIEW

**Comment by zenorocha** [comment]
🎉 🎉 🎉

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ $ yarn start (or npm start)
 
 Use the button below to instantly setup your own Server [google-translate-api](https://github.com/matheuss/google-translate-api) instance on WeDeploy.
 
-[![Deploy](https://avatars3.githubusercontent.com/u/10002920?v=4&s=80)](https://wedeploy.com/docs/intro/using-the-console/)
+[![Deploy](https://cdn.wedeploy.com/images/deploy.svg)](https://console.wedeploy.com/deploy?repo=https://github.com/matheuss/google-translate-api)
```
Comment (brenopolanski) [inline README.md]: The correct would not be: https://console.wedeploy.com/deploy?repo=https://github.com/brenopolanski/server-google-translate-api

?

Comment (jonnilundy) [reply README.md]: Good catch! Just updated.

**Comment by brenopolanski** [comment]
Thanks :+1:

**Pull request merged**